src/migrate.ts

```diff
@@ -190,13 +203,13 @@ function logResult(completedMigrations: Array<Migration>, log: Logger) {
 }
 
 /** Check whether table exists in postgres - http://stackoverflow.com/a/24089729 */
-async function doesTableExist(client: BasicPgClient, tableName: string) {
-  const result = await client.query(SQL`SELECT EXISTS (
-  SELECT 1
-  FROM   pg_catalog.pg_class c
-  WHERE  c.relname = ${tableName}
-  AND    c.relkind = 'r'
-);`)
-
-  return result.rows.length > 0 && result.rows[0].exists
+async function doesTableExist(
+  client: BasicPgClient,
+  migrationTableName: string,
+) {
+  const result = await client.query(SQL`
+    SELECT to_regclass(${migrationTableName}) as matching_tables
```

Comment:
I changed to `to_reclass` as the original query would check if a table name exists in any schema. This meant migrations would be break if a user specified a custom schema in their migration table name.